examples/application/python_editor/python_editor_task.py

```diff
@@ -279,8 +279,9 @@ def create_dock_panes(self):
         """
         browser = PythonBrowserPane()
 
-        def handler():
-            return self.create_editor(browser.selected_file)
```

Comment:
Note: it's not safe to rely on `browser.selected_file` here: that won't always match the path sent to the `activated` event.